discordBot.js

```diff
@@ -1,57 +1,110 @@
-const Discord = require('discord.js');
+const {Client, MessageAttachment} = require('discord.js'),
+      http    = require('http');
```

Comment:
`
const {
  Client,
  MessageAttachment
} = require('discord.js');
const http = require('http');
`
is more readable

Reply:
fixed